src/Components/Pages/ReviewPage.js

```diff
@@ -41,6 +42,12 @@ const ReviewPage = ({
           <DescriptionListTerm>Name</DescriptionListTerm>
           <DescriptionListDescription>{name}</DescriptionListDescription>
         </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>Description</DescriptionListTerm>
+          <DescriptionListDescription>
+            {description || 'Not defined'}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
```

Comment:
Nitpick: I think it could be confusing here to say "Not Defined" on the review page if the description is not filled in. I think maybe the field should be omitted entirely if not filled in? Could be worth asking UX about

Reply:
So I did 'not defined' here as requested by UX our original conversation, it looks like the preferred verbiage here is "No description". I will change this to no description

Reply:
I had just seen in the AC that it was "No description" on the details page. This is about the review page for what data is submitted and I thought it could be confusing to see some description information even when no description is being added to the key.

For example, on this same review page when no Extended Release workload is set, we completely omit the information about that, because it can be confusing to see "Extended release product: no extended release product" when the user didn't provide anything there.

Reply:
Understood I see what you mean probably best to omit like the other fields behave if not entered a value, I did ask UX for verfication on method of action just to make sure.